web/firefox_print_service.js

```diff
@@ -41,6 +41,7 @@ function composePage(
   canvasWrapper.appendChild(canvas);
   printContainer.appendChild(canvasWrapper);
 
+  let currentRenderTask = null;
```

Comment:
Can you add a comment here why we have `currentRenderTask` and  `thisRenderTask`?